notifications/smart_reminder_admin.py

```
@@ -0,0 +1,158 @@
+Admin configuration for smart reminder models
+"""
+from django.contrib import admin
+from .models import SmartReminder, ReminderPattern, ReminderSchedule, ReminderResponse
+
+
+@admin.register(SmartReminder)
+class SmartReminderAdmin(admin.ModelAdmin):
+    list_display = [
+        'title', 'patient', 'reminder_type', 'frequency', 
+        'status', 'priority', 'is_critical', 'created_at'
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)